EssentialApp/EssentialAppTests/FeedUIIntegrationTests.swift

```diff
@@ -317,11 +317,31 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 	
+	func test_feedImageView_imageSelectionTapExecutesImageSelectionBlock() {
+		let image0 = makeImage(url: URL(string: "http://url-0.com")!)
+		let image1 = makeImage(url: URL(string: "http://url-1.com")!)
+		
+		var selectedImages: [FeedImage] = []
+		let (sut, loader) = makeSUT { selectedImages.append($0) }
```

Comment:
It's not clear why the closure is capturing selected images in this context.

In such cases, I recommend using the param label:

```suggestion
		let (sut, loader) = makeSUT(imageSelection: { selectedImages.append($0) })
```